### DISCUSSION TITLE
2.x cpp-tests iOS simulator project

### DISCUSSION THREAD

**Discussion by qoozta** [axmolengine/axmol#1376] [Q&A] [answered]
Using `axmol build -p ios -a x64 -xc "-DAX_USE_ALSOFT=ON" -c` to generate an iOS Simulator xcode project, then try to compile it, it will shows error in libcommon, but the path is openal/common.
`....../axmol/thirdparty/openal/common/althrd_setname.cpp:43:10 'pthread_np.h' file not found`
The arm64 build was no problem, what is wrong with my settings?

**Comment by halx99** [comment, marked as answer]
fixed by https://github.com/axmolengine/axmol/commit/36af496768a3d3cb45a7d2336477c10560a80761
